content/en/agent/kubernetes/operator_configuration.md

```diff
@@ -0,0 +1,214 @@
+| agent.deploymentStrategy.canary.replicas                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| agent.deploymentStrategy.reconcileFrequency                                                                | The reconcile frequency of the ExtendDaemonSet                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| agent.deploymentStrategy.rollingUpdate.maxParallelPodCreation                                              | The maxium number of pods created in parallel. Default value is 250.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| agent.deploymentStrategy.rollingUpdate.maxPodSchedulerFailure                                              | MaxPodSchedulerFailure the maxinum number of not scheduled on its Node due to a scheduler failure: resource constraints. Value can be an absolute number (ex: 5) or a percentage of total number of DaemonSet pods at the start of the update (ex: 10%). Absolute                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
I think this needs rephrased - what does it mean to be the maximum number of not scheduled?